frontend/scripts/game.js

```diff
@@ -25,6 +25,9 @@ function initialiseGame() {
             console.log("Down arrow has been pressed");
             game.moveDown(); 
         }
+        if (event.key === " ") {
+            game.instantDropTetromino();
+        }
```

Comment:
Can you pull main and rebase this on top of it - and update your code with the recent changes?